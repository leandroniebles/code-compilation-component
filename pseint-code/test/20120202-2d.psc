proceso test
   a<-verdadero;
   segun a hacer
      a: escribir 2;
   finsegun
finproceso