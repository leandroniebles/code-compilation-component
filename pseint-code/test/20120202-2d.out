=== Line 3: ExeError 205
